src/pages/404.js

```diff
@@ -0,0 +1,24 @@
+import React from "react";
+
+import Layout from "@components/Layout";
+import TextLink from "@components/TextLink";
+
+const NotFoundPage = () => {
+  return (
+    <Layout title="Page Not Found">
+      <div className="container">
+        <div className="row my-5">
+          <div className="col-12">
+            <h1>Page not found</h1>
+            <p>
+              Oops! That page doesn&apos;t exist. Double check the URL to make sure it&apos;s
+              correct, otherwise you can return to the homepage <TextLink to="/">here</TextLink>.
+            </p>
```

Comment:
Maybe put some sort of graphic on image on this page? It looks a bit bare imo.

Reply:
What do you suggest?

Reply:
I'm not sure. Maybe the animation that's on the homepage?